.github/workflows/ci-cd.yml

```
@@ -30,6 +30,32 @@ jobs:
       - run: pnpm install
       - run: pnpm typecheck
 
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)